ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -40,6 +40,7 @@ import {
 import {isBlank, reactStyles} from 'app/utils';
 import {AnalyticsTracker} from 'app/utils/analytics';
 import {serverConfigStore} from 'app/utils/navigation';
+import {NOT_ENOUGH_CHARACTERS_RESEARCH_DESCRIPTION} from "../../../utils/strings";
```

Comment:
Can you change this to app/utils/strings

Reply:
whoops I didn't lint